examples/scheduler-policy-config-with-extender.json

```diff
@@ -14,15 +14,15 @@
 	{"name" : "ServiceSpreadingPriority", "weight" : 1},
 	{"name" : "EqualPriority", "weight" : 1}
 	],
-"extenders":[ 
+"extender": 
```

Comment:
@RenaudWasTaken not aware of anyone using multiple extenders. Do you have a use case for it? Please see discussion on this PR about getting bind implemented.